includes/admin/class-wc-admin-menus.php

```diff
@@ -151,6 +152,7 @@ public function addons_menu() {
 		/* translators: %s: extensions count */
 		$menu_title = sprintf( __( 'Extensions %s', 'woocommerce' ), $count_html );
 		add_submenu_page( 'woocommerce', __( 'WooCommerce extensions', 'woocommerce' ), $menu_title, 'manage_woocommerce', 'wc-addons', array( $this, 'addons_page' ) );
```

Comment:
Title of this menu needs to be `Marketplace`

Reply:
Renamed to Marketplace.